site/data/addons.yaml

```diff
@@ -792,9 +792,9 @@
   status: alpha
```

Comment:
Off topic. Shall we go beta soon?

Reply:
IMO that's not ready to move to beta.

Reply:
Major tweaks? The current code needs to be tidied up to not rely so much on core or core changed to be properly relied upon.